Automattic/wp-calypso
# Checkout: Move existing-credit-card payment method to calypso

#### Changes proposed in this Pull Request

This is a change extracted from https://github.com/Automattic/wp-calypso/pull/57807 which moves the `createExistingCreditCardMethod()` function from the `@automattic/wpcom-checkout` package into calypso (the `client` directory). This is necessary because the changes in https://github.com/Automattic/wp-calypso/pull/57807 will modify it to make use of components only available in calypso, and moving the file separately will make that PR easier to read.


Co-authored by: @chrisfromthelc 

#### Testing instructions

- Use an account with at least one saved credit card.
- Add a product to your cart and visit checkout.
- Verify that you see the saved payment card in the list of available payment methods.
